storage/range.go

```diff
@@ -2219,6 +2221,37 @@ func (r *Range) ChangeReplicas(changeType proto.ReplicaChangeType, replica proto
 	return nil
 }
 
+// updateRangeDescriptor returns a client.Call to execute a ConditionalPut
+// on the range descriptor.
+func updateRangeDescriptor(descKey proto.Key, oldDesc, newDesc *proto.RangeDescriptor) client.Call {
```

Comment:
Perhaps name this `updateRangeDescriptorCall` to follow the naming of the `client.*Call` functions.